internal/services/v1/debug.go

```diff
@@ -75,6 +96,10 @@ func convertCheckTrace(ctx context.Context, caveatContext map[string]any, ct *di
 		subRelation = ""
 	}
 
+	spiceerrors.DebugAssert(func() bool {
+		return len(ct.Results) == 1
+	}, "multiple results in a single check trace")
```

Comment:
there seems to be an inconsistency in the check trace conversion logic. It's causing the wildcard exclusion test to fail because those operations produce multiple results by definition. Isn't that what we want?

Reply:
Looking

Reply:
Fixed